execute_pipeline.py

```diff
@@ -137,5 +188,31 @@ def _var_replace(in_str, repl_vars):
   return new_str
 
 
+def _load_local_repo(private_repo_root, **pipeline_kwargs):
+  files_dict = {}
+  for root, dirs, files in os.walk(private_repo_root):
+      for fname in files:
+        path = os.path.join(root, fname)
+        rel_path = os.path.relpath(path, private_repo_root)
```

Comment:
Or build the tree of dicts based on the file tree structure. @mvashishtha is doing his version for his git-push PR.